lws_config.h.in

```diff
@@ -61,3 +61,5 @@
 
 /* use SHA1() not internal libwebsockets_SHA1 */
 #cmakedefine LWS_SHA1_USE_OPENSSL_NAME
+
+${LWS_SIZEOFPTR_CODE}
```

Comment:
@JoakimSoderberg wrote:

> Use #cmakedefine LWS_SIZEOFPTR_CODE

Using this doesn't make much sense. It expands to "#define LWS_SIZEOFPTR_CODE" if cmake could determine the size of "void *" (almost always). As a result, _LWS_PAD_SIZE becomes an empty constant and evaluates to 0 in _LWS_PAD() macro.

From [cmake docs](https://cmake.org/cmake/help/v3.0/command/configure_file.html):
> Input file lines of the form “#cmakedefine VAR ...” will be replaced with either “#define VAR ...” or /* #undef VAR */ depending on whether VAR is set in CMake to any value not considered a false constant by the if() command.

On the other hand, [CHECK_TYPE_SIZE()](https://cmake.org/cmake/help/v3.0/module/CheckTypeSize.html) allows one to generate preprocessor code that expands to a number, even for an OS X multi-architecture binary. See the documentation for this macro.